### PR TITLE
capitalisation and spacing

### LINKED ISSUE #5102: Fix spacing & capitalization in dashboard activity
## First Time?

This is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

## The problem

Our UI under our Dashboard Activity section needs a little improvement.

Screen Shot 2019-03-16 at 1 20 32 AM

## Solution

Let's make it look nice and consistent by adding spacing & capitalization, like this:

Screen Shot 2019-03-16 at 6 40 50 AM

Where to find the relevant lines of code: 
1) The spacing issue. Add a space after the | (pipe symbol)
https://github.com/publiclab/plots2/blob/d083415283dd89e0644e5bfe5ceb8e30b75dbecf/app/views/dashboard/_activity.html.erb#L42

2) The capitalization issue. Capitalize the first word **of the string**. You do not need to change the key. 
https://github.com/publiclab/plots2/blob/d083415283dd89e0644e5bfe5ceb8e30b75dbecf/config/locales/en.yml#L287

Thanks!!

## Bonus
I am not married to the way the text is laid out in general. If you have any additional changes you want to make, add them on to your PR and thank you! 

## Step by Step
- [ ] Claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ] Fork the repository and set it up locally following the main repo README instructions [https://github.com/publiclab/plots2](https://github.com/publiclab/plots2)
 - [ ] Create a new feature branch with a unique name descriptive to the issue
- [ ] Try to fix the issue following the steps above, but even before you're done, you can:
  commit your changes to your branch and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] Reference this issue in your pull request body
- [ ] Once you submit your pull request, an additional checklist will be provided for getting it merged

### Please keep us updated
💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

##  💬   Get help
If you need any help - here are some options: 

- Comment below
- Join our gitter chat at https://gitter.im/publiclab/publiclab

### PR DESCRIPTION
**Pull request by kuja24**
Fixes #5102  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by kuja24** [comment]
@ananyaarun Sure !! I'll recheck and update.

**Comment by kuja24** [comment]
@ananyaarun I got a green tick.So how do I check the logs of what needs to be updated?

**Comment by ananyaarun** [comment]
@kuja24 , green tick means travis CI tests passed :) U don't need to check any logs for updating.
The issue asks you to capitalize the 287th line in the en.yml file, But i don't see that made in the changes, instead i see a lot of other lines capitalized. To solve this issue U'll have to capitalize only the 287th line of the file Thanks !

**Comment by kuja24** [comment]
> @kuja24 , green tick means travis CI tests passed :) U don't need to check any logs for updating.
> The issue asks you to capitalize the 287th line in the en.yml file, But i don't see that made in the changes, instead i see a lot of other lines capitalized. To solve this issue U'll have to capitalize only the 287th line of the file Thanks !

**Comment by kuja24** [comment]
@ananyaarun I am making the changes i.e capitalizing line287. Do in need to uncapitalize other changes that I have made. Also I need to open a new pull request after making these changes , right? Please let me know if I need to change anything else also.
Thanks!

**Comment by ananyaarun** [comment]
@kuja24 , No need to close this PR, U can make the changes here itself :)
Just capitalize the 287th line for now, I am not sure if others need to be.
Great job so far :smiley:

**Comment by kuja24** [comment]
@ananyaarun Please check if it can be merged now.

**Comment by kuja24** [comment]
@ananyaarun I have undone the changes.Also, I did those changes because I thought it would improve the UI (the capitalization part).So , if you want you can consider that also :).
Otherwise , I have undone the changes.You can check and merge.
Thanks!

File: config/locales/en.yml [hunk 1]
```diff
@@ -284,7 +284,7 @@ en:
         events: "Events"
         comments: "Comments"
         wiki: "Wiki"
-      from_other: "from other"
+      from_other: "From other"
```
Comment (gauravano) [inline config/locales/en.yml]: hey @kuja24, only change in this line is required. please undo other changes. Thanks!

**Comment by kuja24** [comment]
@gauravano I have already undone the changes. You can check them in the commits.And please let me know if any other change is required.

**Comment by kuja24** [comment]
@gauravano Have you checked the changes?

**Comment by gauravano** [comment]
Hey @kuja24, in your commit you haven't undone changes completely https://github.com/publiclab/plots2/pull/5290/commits/39312892913d1a53364dc464277101e1e9e4a1ae

you de-capitalized only first letter of the first word but not remaining words. you can also see in the changes :point_down: 
Screenshot from 2019-03-31 17-06-28

**Comment by sashadev-sky** [comment]
@kuja24 
Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by sashadev-sky** [comment]
@kuja24 I am closing this PR because we haven't gotten a response back in a while. I am solving it instead myself in PR #5421 as I am already editing this specific page. Sorry that you did not get to complete it -- please feel free to ask for a new FTO anytime :)

**Pull request closed without merging**